src/Processors/QtiV2/In/RubricBlockMapper.php

```diff
@@ -82,17 +82,10 @@ public function parseWithRubricBlockComponent(RubricBlock $rubricBlock, $foundSc
                     $mapper = new DistractorRationaleResponseMapper();
                     $distractorRationalResponseLevelArray = $mapper->parseWithDistractorRationaleResponseComponent($rubricBlock);
                     $result ['question_metadata'] = $distractorRationalResponseLevelArray;
-                } else if ($views->contains(View::AUTHOR)) {
-                    $result = [
-                        'question_metadata' => [
-                            'distractor_rationale_author' => [
-                                [
-                                    'label' => $rubricBlock->getLabel(),
-                                    'content' => QtiMarshallerUtil::marshallCollection($rubricBlock->getContent()),
-                                ]
-                            ],
-                        ],
-                    ];
+                } else if ($views->contains(View::AUTHOR) || $views->contains(View::SCORER) || $views->contains(View::TUTOR)) {
```

Comment:
This is a breaking change @rakeshqdsa (FYI @michaelsharman). We have users of this library that require support for content in `distractor_rationale_author` via the `View::AUTHOR` flag. Removing this code will prevent that use case. I'd suggest we should be separating the handling of the AUTHOR role from the SCORER/TUTOR roles.

Reply:
As per WBL-83
When “use” is “rationale”, and “view” contains either “author”, “scorer” or “tutor”, then write to metadata.distractor_rationale_per_response as an array of strings.
You mean to say when use is "rationale" and view contain "author" we should use distractor_rationale_author and for scorer/turor we should use distractor_rationale_per_response .
Please confirm

Reply:
Correct, that is my assessment of it, but it sounds like this is contradicting the spec. Will need @michaelsharman to confirm here. All I know is this will break existing usage patterns for AUTHOR roles. It also makes sense to me that if the view is supposed to be AUTHOR, then it should be visible primarily in the authoring applications (Author and QE), rather than the assessment applications (which `distractor_rationale_per_response` is designed for).

Reply:
Done as per suggestion.

Reply:
This is getting closer - you'll also need to either move or remove this line (the case with `class="DistractorRationale"`):
https://github.com/Learnosity/learnosity-qti/pull/75/files#diff-daa88880ab5ffffbfd900c0222d53964L78

This is because the cases that use `distractor_rationale_author` sometimes apply `class="DistractorRationale"` to the `rubricBlock`, which means the case below will never be reached. Let me know whether that makes sense.

Reply:
The link above wasn't clear, sorry. It's pointing at line 78 in this class.

Reply:
This is a small change so I've just gone ahead and applied the fix.